### ISSUE TITLE
Hotbar turns gray(in 2nd or 3rd) person with OptiFine

### ISSUE THREAD

**Issue by THEREALWWEFAN231** [cabaletta/baritone#1635]
## Some information
Operating system: Windows 10
Java version: I think Minecraft preinstalled version 1.8.0_51
Minecraft version: 1.12.2 forge
Baritone version: baritone-standalone-forge-1.2.14
Forge mods (if used): OptiFine_1.12.2_HD_U_F5

## How to reproduce
Go into a single player world, hit F5(or your key to toggle perspective) transparentish hotbar.

## Other information
Only happens with OptiFine, other people have the same issue.

**Comment by dominikaaaa** [comment]
Uninstall optifine, that should fix the issue.

**Comment by THEREALWWEFAN231** [comment]
No crap lol, but I'm pretty sure they(the developers) want to keep baritone compatible with optifine, considering a lot of people use OptiFine and this screenshot 
![image](https://user-images.githubusercontent.com/21975901/80547937-51342080-896e-11ea-966a-189c3b1d47aa.png) maybe not I don't know.

**Comment by dominikaaaa** [comment]
Oh I didn't realise this was the baritone repo, I thought it was the impact repo. In that case, Optifine is important.

**Comment by dominikaaaa** [comment]
I can't reproduce this issue on Linux, can you send a video?

**Comment by THEREALWWEFAN231** [comment]
https://streamable.com/gjvhre

**Comment by dominikaaaa** [comment]
Now that I think about it, this *might* be baritones fault. I thought it was my own shitty open gl but I did always have baritone installed when testing...

**Comment by 0-x-2-2** [comment]
Can you reproduce with http://optifine.net/adloadx?f=preview_OptiFine_1.12.2_HD_U_F6_pre1.jar or an older Baritone build?

**Comment by THEREALWWEFAN231** [comment]
Yes it still happens with that prebuild version, it starts at baritone version 1.2.10, it does not occur in 1.2.9.

**Comment by UltraBlackLinux** [comment]
@dominikaaaa im litterally having the same problem in linux. Already wondered what that was.
